include/bbp/sonata/nodes.h

```diff
@@ -44,6 +44,8 @@ class SONATA_API NodePopulation: public Population
      *
      * Note: This does not match Dynamics_params datasets
      */
+    std::vector<NodeID> nodeIDs(const Selection& selection) const;
```

Comment:
I think the comment note above relates to the function below.

Reply:
Whoo my bad.